### DISCUSSION TITLE
Importing from Reddit CSV Export

### DISCUSSION THREAD

**Discussion by kevin96246** [shadowmoose/RedditDownloader#276] [Q&A]
Hi,
I downloaded my reddit history from Reddit. i wonder if I can import "saved_posts.csv" and exactly how should I do that. The [doc ](https://shadowmoose.github.io/RedditDownloader/Advanced_Usage/Settings/#importing-from-reddit-csv-export) says I can only import comment_votes.csv & post_votes.csv. In addition, when trying to import these two files, terminal says "Exception: Unable to connect to pushshift.io. Max retries exceeded." It also tried to use PushShift on WebUI and the same issue appear (unable to connect to pushshift). How should i resolve the issue? I also checked that I did have pushshift api installed.

**Comment by GeraldPreston** [comment]
I have the same question - I've tried to get it to read the saved_posts.csv file but no luck so far! Not sure if its not possible or I'm not configuring it correctly. Would love any advice! RMD is a fantastic tool otherwise, I'd just like to know that it's hitting all my saved posts, rather than just the shortened list that the API returns (and soon won't!)

**Comment by shadowmoose** [comment]
Unfortunately, the CSV import relied on PushShift, which is no longer operational. Thank Reddit for that. See [this surprisingly prophetic post](https://github.com/shadowmoose/RedditDownloader/discussions/289) from earlier for more information.

**Reply by GeraldPreston** [reply]
Thanks for the info!
